addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -22,6 +23,17 @@ weSnippetEditor.Class.include({
         THEME: 'theme',
     }),
 
+    /**
+     * @override
+     *
+     * @param {Widget} parent
+     * @param {object} options
```

Comment:
No blank lines between `@aaa` but in this case the `@param` are not needed: when you use `@override` you don't have to re-document every parameter. In fact as it is a constructor here, we don't even use `@override` but `@constructor`... but not really important.